### DISCUSSION TITLE
Support udtf

### DISCUSSION THREAD

**Discussion by smallzhongfeng** [databendlabs/databend#13444] [Q&A] [answered]
I seen some issue like https://github.com/datafuselabs/databend/issues/5731, but it has not been completed yet. I would like to ask if the community still plans to support udtf on distributed systems.

**Comment by bohutang** [comment, marked as answer]
Now, the better way to do is external function: https://databend.rs/doc/sql-commands/ddl/external-function/

**Reply by smallzhongfeng** [reply]
So it can implement custom functions such as udf, udtf, and udaf, right? @BohuTANG

**Reply by bohutang** [reply]
Yes.
